src/utils.jl

```diff
@@ -0,0 +1,12 @@
+"""
+    _as_svec(x)
+
+Accepts a single object, an `SVector` of objects or a tuple of objects, and returns them in a single "layer" of SVector.
+
+Useful with [`base_state`](@ref).
+"""
+function _as_svec end
```

Comment:
It's only used in QEDcore, and it's also only used in tests there, specifically the `base_states` tests. It might still make sense to keep it in QEDbase though, since it is mentioned there as helpful in conjunction with the base_states. This could also be deemed an implementation detail. I don't really have a strong opinion on it.

It could be argued anyways that some of the documentation on the `base_state` interface is now on implementation details that would only apply in QEDcore.

I have opened #75